src/interface/api-inbound.yaml

```diff
@@ -323,6 +323,147 @@ paths:
       operationId: ThirdpartyRequestsTransactionsPost
       summary: example get
       description: The HTTP request GET /hello is used to return some example json.
+  '/thirdpartyRequests/transactions/{ID}/authorizations':
+    put:
+      operationId: UpdateThirdpartyAuthorization
+      summary: UpdateThirdpartyAuthorization
+      description: |
+        The HTTP request `PUT /thirdpartyRequests/transactions/{id}/authorizations` is used by the auth-service to update a thirdparty authorization after successful validation.
+        For an unsuccessful authorization result, the `PUT /thirdpartyRequests/transactions/{id}/authorizations/error` will be called by the auth-service, instead of this endpoint.
+      parameters:
+        - name: ID
+          in: path
+          required: true
+          schema:
+            type: string
+          description: The identifier value.
+        - name: Content-Length
+          in: header
+          required: false
+          schema:
+            type: integer
+          description: |
+            The `Content-Length` header field indicates the anticipated size of the
```

Comment:
maybe it is better to use headers from api-snippets: https://github.com/mojaloop/api-snippets/tree/master/v1.0/openapi3/headers
and add to api-snipptes what is needed or create a story in the backlog to do it?

Reply:
Ok